backend/services/syson-services/pom.xml

```diff
@@ -68,6 +68,11 @@
 			<artifactId>sirius-components-collaborative-diagrams</artifactId>
 			<version>${sirius.web.version}</version>
 		</dependency>
+		<dependency>
```

Comment:
Why this new dependency?

Reply:
I had a compilation error to something that was in that project.
I don't know what has changed since, but it seems to work without this dependency, I will remove it.